tools/benchcomp/benchcomp/visualizers/__init__.py

```diff
@@ -12,8 +14,42 @@
 import benchcomp.visualizers.utils as viz_utils
 
 
-# TODO The doc comment should appear in the help output, which should list all
-# available checks.
+
+@dataclasses.dataclass
+class run_command:
+    """Run an executable command, passing the result as a JSON file on stdin.
+
+    This allows you to write your own visualization, which reads a result file
```

Comment:
Where can one find the format of the result file?

I see the following sentence on [this page ](https://model-checking.github.io/kani/benchcomp-conf.html), but I don't see any schema.

> This page describes the file's schema and lists the different parsers and visualizations that are available.

Reply:
Yeah, not yet merged, sorry :( I have a half-written schema and was going to generate the documentation from it, similar to how the visualization docs are generated.

For now if you'd like to use this feature, I suggest starting off by setting
```yaml
type: run_command
command: cat - > /tmp/result.yaml
```
which just takes stdin and redirects it to a file, so you can then open and read that file to see the structure.

Reply:
Can you please remove the statement I pasted until then?

Reply:
Done